django/locale/translation/translations.json

```diff
@@ -3982,5 +3982,13 @@
     "An error occurred.": {
         "fr": "",
         "fr_auto": "Une erreur s’est produite."
+    },
+    "Read each document separately  (slow, $$)": {
+        "fr": "",
+        "fr_auto": "Lisez chaque document séparément (lent, $$)"
+    },
+    "Read all documents at once (slowest, $$)": {
```

Comment:
I suggest slightly different wording here. Because the per-document responses stream "in parallel", they appear to all get read/answered "at once". Maybe the following wording is more clear?

"Answer for each document separately (slow, $$)"
"Compare all documents in single answer (slowest, $$)"

Reply:
Good point. I wanted to make it clear that the documents are read in their entirety. I'll try to think of a wording that captures both (while remaining succinct).